config/settings.d/dns_nsupdate.yml.example

```diff
@@ -0,0 +1,16 @@
+---
+#
+# Configuration file for 'nsupdate' dns provider
+#
+# Can be true, false
+:enabled: true
+
+#:dns_key: /etc/rndc.key
+# use this setting if you are managing a dns server which is not localhost though this proxy
+#:dns_server: dns.domain.com
+# use this setting if you want to override default TTL setting (86400)
+#:dns_ttl: 86400
```

Comment:
* Agree re: dns_ttl, and it's an easy fix.
* I think I'd actually prefer to keep dns_provider setting, but get rid of enable/disable in providers: this way no need to handle conflicts between providers.
* Will take a look at the migration issue.